test/serialize.test.ts

```diff
@@ -224,6 +232,104 @@ describe("serialize", () => {
     });
   });
 
+  describe("sfn", () => {
+    const stack = new Stack();
+    const sfn = new StepFunction(stack, "id", () => {});
+
+    test("sfn", async () => {
+      const [srlz] = await serialize(
+        () => () => {
+          return sfn({ input: {} });
+        },
+        []
+      );
+      expect(srlz).toMatchSnapshot();
+
+      const bundled = await bundle(srlz);
+      expect(bundled.text).toMatchSnapshot();
+      expect(bundled.text).toHaveLengthLessThan(BUNDLED_MAX_SIZE);
```

Comment:
Should we do this per test instead of capping each test at the same amount?

Reply:
I set 50k as the max and then up them as needed. Could try lower and see what breaks.